script/run_generate.sh

```diff
@@ -72,10 +72,10 @@ fi
 mkdir -p build/
 cd build/
 
-"${ALPAKA_CI_CMAKE_EXECUTABLE}" -G "${ALPAKA_CI_CMAKE_GENERATOR}" ${ALPAKA_CI_CMAKE_GENERATOR_PLATFORM} \
+"${ALPAKA_CI_CMAKE_EXECUTABLE}" --verbose -G "${ALPAKA_CI_CMAKE_GENERATOR}" ${ALPAKA_CI_CMAKE_GENERATOR_PLATFORM}\
```

Comment:
Adding this flag feels not necessary for this PR to work. Is this a functional change? Or purely more output from cmake?

Reply:
@BenjaminW3 Could you check this question so that I know if I can merge this PR.

Reply:
Ah, I overlooked this. This change was part of @j-stephan s initial PR for first class CUDA support with CMake because there we lose the option to improve the debug output only for the `FindCUDA` script.
I found it useful for investigation here even with the CUDA change so I added it here.